models/admin.go

```diff
@@ -18,25 +18,27 @@ import (
 	"code.gitea.io/gitea/modules/setting"
 )
 
-type NoticeType int
+type noticeType int
```

Comment:
I'd keep this public, since it's used on a public field of a public struct below (`Notice`).

:+1:  for the other changes.